pkg/storage/flags.go

```diff
@@ -6,8 +6,15 @@ import (
 	"github.com/spf13/pflag"
 )
 
+const (
+	// flagSendCredentialOption specify whether to send credentials to tikv
+	flagSendCredentialOption = "send-credentials-to-tikv"
+)
+
 // DefineFlags adds flags to the flag set corresponding to all backend options.
 func DefineFlags(flags *pflag.FlagSet) {
+	flags.BoolP(flagSendCredentialOption, "c", true,
```

Comment:
Should it be moved into storage/s3.go? What do you think @kennytm ？

Reply:
gcs, azure or other cloud vendors may also need this.